### ISSUE TITLE
Add list of webhooks on developers page

### ISSUE THREAD

**Issue by FelixMalfait** [twentyhq/twenty#3271]
See [Figma](https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=13273%3A92445&mode=dev)
![Settings - API](https://github.com/twentyhq/twenty/assets/6399865/f60d477e-2565-4c8f-9a47-d28057a59d65)

We already have the section for to list API keys and want to add the list of webhooks below. 
Everything on the backend has been done already so it should be a purely frontend ticket.
Note: there's a field called "operation", let's ignore it for now and just do as displayed in the design.

I will create a separate issue for the create webhook page.

**Comment by lakshay-saini-au8** [comment]
@FelixMalfait Can I work on this and the follow up issue

**Comment by lakshay-saini-au8** [comment]
Also, I checked the API page it doesn't look like what we have in Figma.
![Screenshot 2024-01-06 at 5 29 27 PM](https://github.com/twentyhq/twenty/assets/19304527/95a12ed3-8299-4f3b-8409-6f4e1b7862a0)

**Comment by FelixMalfait** [comment]
@lakshay-saini-au8 yes indeed, part of this issue is to move the "create key" button and change the page title, on top of adding the webhook section. Thanks a lot! Assigning you

**Comment by FelixMalfait** [comment]
Hey @lakshay-saini-au8 any progress on this yet? Sorry don't mean to add pressure but we would love to release this soon!

**Comment by lakshay-saini-au8** [comment]
@FelixMalfait yes I am working will raise the PR soon
